docs/UserMenu.md

```diff
@@ -74,14 +74,21 @@ const avatar = <Avatar><SendIcon/></Avatar>;
 
 Any other props supplied will be provided to the root element (`div`).
 
-### Classes
+### Style Overrides
 
-You can override the classes used by Brightlayer UI by passing a `classes` prop. It supports the following keys:
+You can override the default styles used by Brightlayer UI by:
 
-| Name        | Description                               |
-| ----------- | ----------------------------------------- |
-| root        | Styles applied to the root element        |
-| bottomSheet | Styles applied to responsive bottom sheet |
+-   using the `sx` prop
+-   passing a `classes` prop with keys from the `Name` column below
+-   using the `Global CSS Class` in your main stylesheet
+
+For more details on styling options check out our [Styling Guide](https://github.com/brightlayer-ui/react-component-library/tree/master/docs#style-guide).
+
+| Name        | Global CSS Class         | Description                               |
+| ----------- | ------------------------ | ----------------------------------------- |
+| root        | .BluiUserMenu-root       | Styles applied to the root element        |
+|             | .BluiUserMenu-avatarRoot | Styles applied to the avatarRoot element  |
+| bottomSheet | n/a                      | Styles applied to responsive bottom sheet |
```

Comment:
This should still have a global class, but should have a callout saying that this element lives in a separate container and can't be styled via sx nesting rules.